openfisca_core/tracers.py

```diff
@@ -130,53 +162,125 @@ def _get_time_in_sec(self) -> float:
     def print_computation_log(self, aggregate = False):
         self.computation_log.print_log(aggregate)
 
-    def generate_performance_graph(self, dir_path):
+    def generate_performance_graph(self, dir_path: str) -> None:
         self.performance_log.generate_graph(dir_path)
 
+    def generate_performance_tables(self, dir_path: str) -> None:
+        flat_trace = self.get_flat_trace()
+
+        csv_rows = [
+            {'name': key, 'calculation_time': trace['calculation_time'], 'formula_time': trace['formula_time']}
+            for key, trace in flat_trace.items()
+            ]
+        self._write_csv(os.path.join(dir_path, 'performance_table.csv'), csv_rows)
+
+        aggregated_csv_rows = [
+            {'name': key, **aggregated_time}
+            for key, aggregated_time in self.aggregate_calculation_times(flat_trace).items()
+            ]
+
+        self._write_csv(os.path.join(dir_path, 'aggregated_performance_table.csv'), aggregated_csv_rows)
+
+    def generate_performance_data(self, dir_path: str) -> None:
+        self.generate_performance_graph(dir_path)
+        self.generate_performance_tables(dir_path)
+
+    def _write_csv(self, path: str, rows: List[Dict[str, Any]]) -> None:
+        fieldnames = list(rows[0].keys())
+        with open(path, 'w') as csv_file:
+            writer = csv.DictWriter(csv_file, fieldnames = fieldnames)
+            writer.writeheader()
+            for row in rows:
+                writer.writerow(row)
+
+    def aggregate_calculation_times(self, flat_trace: Dict) -> Dict[str, Dict]:
+
+        def _aggregate_calculations(calculations):
+            nof_calculations = len(calculations)
```

Comment:
I'm not a fan of `nof_` as an abbreviation. `nb_calculations` is more standard, or `count_calculations'.

Reply:
I used to think the same, until I recently learned from native English speakers that `nb` is [**not** an abbreviation for number in English](https://forum.wordreference.com/threads/abbreviation-of-number-n-n%C2%B0-nr-nbr-no.264328/), but only for _nota bene_.
"No" is the standard abbreviation, but `no_calculations` would be awfully confusing. 

I'm ok with switching to `calculation_count` though